style.css

```diff
@@ -147,54 +156,266 @@
   opacity: 0.3;
 }
 
-.tonic {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  color: rgb(0, 0, 82);
-  margin: 15px;
-  margin-top: 30px;
+.project-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  background-color: white;
+  box-sizing: border-box;
+  border-radius: 10px;
+  padding: 10px;
+}
+
+.work {
+  background-color: #e5e5e5;
+  width: 100%;
 }
 
-.tonic-img {
-  margin-left: 27px;
+.work-img {
   display: flex;
   flex-direction: column;
   justify-content: center;
-  height: 370px;
-  width: 370px;
+  margin: 10px 10px;
 }
 
-.tonic-text {
+.project-title {
+  margin: 5px 10px;
+  display: flex;
   flex-direction: column;
+  align-items: flex-start;
+  font-size: 42px;
+  color: rgb(0, 10, 47);
+  font-family: "Poppins", sans-serif;
+  font-weight: bold;
+  margin-top: 12px;
+  margin-left: 15;
+}
+
+.some-list {
+  margin: 5px 10px;
+  margin-left: 15px;
+}
+
+.work-info {
+  display: flex;
+  flex-direction: row;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  padding: 15px;
+}
+
+.lang-list {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 82%;
+  margin-left: -5%;
+  padding: 4px 12px;
+}
+
+.Html {
+  margin: 12px 2px;
+  padding: 5px 12px;
+  font-size: 16px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border: 2px solid #ebebff;
+  border-radius: 9px;
+  outline: none;
+}
+
+.Css {
+  margin: 12px 2px;
+  padding: 5px 10px;
+  font-size: 16px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border: 2px solid #ebebff;
+  border-radius: 9px;
+  outline: none;
+}
+
+.js {
+  margin: 12px 2px;
+  padding: 5px 10px;
+  font-size: 20px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border: 2px solid #ebebff;
+  border-radius: 9px;
+  outline: none;
+}
+
+.button {
+  margin: 1.3rem;
+  padding: 0.7rem 1.5rem;
+  font-size: 20px;
+  color: #6070ff;
+  background-color: #fff;
+  border: 2px solid blue;
+  border-radius: 7px;
+  outline: none;
+  cursor: pointer;
+}
+
+.button:hover {
+  opacity: 0.7;
+}
+
+.button:active {
+  opacity: 0.3;
+}
```

Comment:
- [ ] I think it would be better to follow the button styling: **Active, Hover** and **Pressed** as suggested by the previous code reviewer for the best user experience 😍

![image](https://user-images.githubusercontent.com/102757126/212393393-b6426fdc-9fd9-4a42-869a-fb0f3bf204a7.png)

You may kindly follow this[ link ](https://www.freecodecamp.org/news/css-button-style-hover-color-and-background/)on CSS Button Style – Hover, Color, and Background for more info🙏